### PR TITLE
Stylesheets update and new Darker Stylesheet

### PR DESCRIPTION
**Pull request by pgilfernandez**
Update to v2.20 for Dark and Light stylesheets:
- minor fixes
- new background icon

Added a brand new **Darker** stylesheet

### CODE REVIEW

File: src/Gui/Stylesheets/Dark-blue.qss [hunk 1]
```diff
@@ -497,7 +498,7 @@ QTabBar {
     background-color: transparent;
 }
 
-/* Workaround for QTabBars created from docked QDockWidgets which don't draw the border if not set and reset as follows: */
+/* Workaround for QTabBars created from docked QDockWidgets which don't draw the border if not set and reseted as follows: */
```
Comment (vocx-fc) [inline src/Gui/Stylesheets/Dark-blue.qss]: Just a heads up, in English, the past and participle of "set" is "set". Same with "reset".

Comment (pgilfernandez) [reply src/Gui/Stylesheets/Dark-blue.qss]: Thanks for the fix =)

Comment (vocx-fc) [reply src/Gui/Stylesheets/Dark-blue.qss]: I didn't change anything. What I mean is, you have to change "reseted" to "reset" :)

Comment (pgilfernandez) [reply src/Gui/Stylesheets/Dark-blue.qss]: OK, I thought you already included the fix.
I'll do it then, of course.
Thanks again

**Comment by pgilfernandez** [comment]
Fixed, ready to merge?

**Pull request merged**